scripts/bash/gitlab_install.sh

```diff
@@ -15,35 +15,41 @@ fi
 # Install Dependencies
```

Comment:
Your base assumption here is happy flow.
Adding some conditions for fail commands?

Reply:
We have tasks for this script to use functions, conditions and logging for debugging purposes.
No integrated in this version